matrix/config.py

```diff
@@ -629,6 +629,24 @@ def __init__(self):
                 ("Reconnect to the server if the lag is greater than this "
                  "value (in seconds)"),
             ),
+            Option(
+                "autoreconnect_delay_growing",
+                "integer",
+                "",
+                1,
+                17,
+                "2",
+                ("Reconnect backoff exponent"),
```

Comment:
We could probably expand the help message a bit to be more clear, the IRC plugin can be used as inspiration again.